commands/copy.go

```diff
@@ -74,6 +78,21 @@ func (r *copyRunner) run(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+// readNoMoreThan reads at most, max bytes from reader.
```

Comment:
Also, could you address some annotations from the linter?

Reply:
Yes yes, I did see the failures and will address them